sdk/tests/deqp/framework/opengl/gluShaderProgram.js

```diff
@@ -155,9 +155,9 @@ gluShaderProgram.Shader = function(gl, type) {
  * @constructor
  */
 gluShaderProgram.ProgramInfo = function() {
-    /** @type {string} */ var infoLog;
-    /** @type {boolean} */ var linkOk = false;
-    /** @type {number} */ var linkTimeUs = 0;
+    /** @type {string} */  this.infoLog = '';
```

Comment:
Can you also update the annotation to align with others? say, gluShaderProgram.ShaderInfo